packages/pwa/src/components/feedSubscriptions/FeedSubscriptionSettings.tsx

```
@@ -202,6 +203,12 @@
   );
 };
 
+const FeedSubscriptionIntervalSetting: React.FC<{
```

ESLint

[error] 'FeedSubscriptionIntervalSetting' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)